jboss/container/config/cd/19.0/standalone-openshift.xml

```diff
@@ -15,7 +15,6 @@
         <extension module="org.jboss.as.jsf"/>
         <extension module="org.jboss.as.logging"/>
         <extension module="org.jboss.as.mail"/>
-        <extension module="org.jboss.as.modcluster"/>
```

Comment:
We didn't remove the modcluster in cd19. We should keep it in the reference file.